pandas/io/sql.py

```diff
@@ -139,7 +139,7 @@ def execute(sql, con, cur=None, params=None):
     ----------
     sql : string
         Query to be executed
-    con : SQLAlchemy engine or sqlite3 DBAPI2 connection
+    con : SQLAlchemy connectable or sqlite3 DBAPI2 connection
```

Comment:
I worry a bit that the term 'connectable' is not really known to most users? Maybe just use `engine/connection`?